File: src/com/sdk/words/demo/ui/accurategeneral/AccurateGeneralWindow.java
package com.sdk.words.demo.ui.accurategeneral;

import com.xiuye.window.FxmlWindow;

public class AccurateGeneralWindow {

	public static void main(String[] args) {
//		System.out.println(WordsWindow.class.getResource("app.fxml"));
		FxmlWindow.running("通用文字识别（含位置高精度版）",AccurateGeneralWindow.class.getResource("accurateGeneral.fxml"));
	}

}

File: src/com/sdk/words/demo/ui/accurategeneral/AccurateGeneralWindow.js
var cls = Java.type("com.sdk.words.demo.ui.accurategeneral.AccurateGeneralWindow");
var sys = Java.type("java.lang.System");

print(sys.getProperty("java.class.path"));

//console.log(123);

for(var key in cls){
	print("key="+key);
}

cls.main([]);
print("end");
